pallets/api/src/fungibles/tests.rs

```diff
@@ -62,32 +108,69 @@ fn approve_works() {
 		let owner = ALICE;
 		let spender = BOB;
 
-		create_token_and_mint_to(ALICE, token, ALICE, value);
-		assert_eq!(0, Assets::allowance(token, &ALICE, &BOB));
-		assert_ok!(Fungibles::approve(signed(ALICE), token, BOB, value));
-		assert_eq!(Assets::allowance(token, &ALICE, &BOB), value);
+		// Approves a value to spend.
```

Comment:
This test is ~70 lines, could really be split out into different tests to make it easier to review/understand/maintain.